inc/classes/optimization/JS/class-abstract-js-optimization.php

```diff
@@ -93,7 +93,7 @@ protected function is_minify_excluded_file( $tag ) {
 			return true;
 		}
 
-		$file_path = rocket_extract_url_component( $tag[2], PHP_URL_PATH );
```

Comment:
Same comments here about type hinting and guarding in case `$tag` doesn't have 3 elements.

Reply:
Done in https://github.com/wp-media/wp-rocket/commit/4093a46af1c2cb5a62512f4e6f5f29965c363365